packages/sdk/src/across/clients/bridgePool.ts

```diff
@@ -83,13 +84,17 @@ class PoolState {
   public async read(latestBlock: number, previousBlock?: number) {
     if (this.l1Token === undefined) this.l1Token = await this.contract.l1Token();
     // typechain does not have complete types for call options, so we have to cast blockTag to any
-    const exchangeRatePrevious = await this.contract.callStatic.exchangeRateCurrent({
-      blockTag: previousBlock || latestBlock - 1,
-    } as any);
+    const [exchangeRatePrevious, liquidityUtilizationCurrent] = await Promise.all([
+      this.contract.callStatic.exchangeRateCurrent({
+        blockTag: previousBlock || latestBlock - 1,
+      } as any),
+      this.contract.callStatic.liquidityUtilizationCurrent(),
+    ]);
```

Comment:
My question is which clients perform these calls because they require gas to be paid

Reply:
What do you mean?

It's not read-only, but that's just a quirk of how the contract works. I think your callStatic accounts for this, right?

Reply:
Yep, you're right